requirements-docs.txt

```
@@ -1,19 +1,20 @@
-mike
+griffe==0.42.1
+mike==2.0.0
 markdown-captions
 mkdocs==1.5.3
 mkdocstrings[python]>=0.24
```

For some reason, in my environment this fails if I don't pin the version of `mkdocstrings-python` to a newer version (See related [issue](https://github.com/mkdocstrings/griffe/issues/197))

```suggestion
mkdocstrings[python]>=0.24
mkdocstrings-python==1.9.0
```